### DISCUSSION TITLE
I am not getting verification code

### DISCUSSION THREAD

**Discussion by shorty-20** [community/community#119417] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to login to the account but I am not receiving the verification code for login in my desktop .

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @shorty-20 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by CopperEagle** [comment]
It should work again, according to the github status page: https://www.githubstatus.com/incidents/b6f4j9ljz9r6
Do you still have problems receiving the verification code email?
